chat-android/src/main/java/com/ably/chat/Messages.kt

```diff
@@ -32,18 +32,26 @@ interface Messages : EmitsDiscontinuities {
     /**
      * Subscribe to new messages in this chat room.
      * @param listener callback that will be called
-     * @returns A response object that allows you to control the subscription.
+     * @return A response object that allows you to control the subscription.
      */
     fun subscribe(listener: Listener): MessagesSubscription
 
     /**
      * Get messages that have been previously sent to the chat room, based on the provided options.
      *
-     * @param options Options for the query.
-     * @returns A promise that resolves with the paginated result of messages. This paginated result can
-     * be used to fetch more messages if available.
+     * @param start The start of the time window to query from. See [QueryOptions.start]
+     * @param end The end of the time window to query from. See [QueryOptions.end]
+     * @param limit The maximum number of messages to return in the response. See [QueryOptions.limit]
+     * @param orderBy The order of messages in the query result. See [QueryOptions.orderBy]
+     *
+     * @return Paginated result of messages. This paginated result can be used to fetch more messages if available.
      */
-    suspend fun get(options: QueryOptions): PaginatedResult<Message>
+    suspend fun get(
+        start: Long? = null,
```

Comment:
As per [chat-js QueryOptions](https://github.com/ably/ably-chat-js/blob/e8380583424a83f7151405cc0716e01302295eb6/src/core/messages.ts#L45), both `start` and `end` has default values

Reply:
Also `orderBy` should be `OldestFirst` since default value for the same is mapped to `forwards`

Reply:
Discussed on the standup, default should be backwards